src/python/grpcio/grpc/_channel.py

```diff
@@ -20,13 +20,22 @@
 import sys
 import threading
 import time
-
-import grpc
-from grpc import _common
-from grpc import _compression
-from grpc import _grpcio_metadata
+import types
+from typing import (Any, Callable, Iterable, Iterator, Optional, Sequence, Set,
+                    Tuple, Union)
+
+import grpc  # pytype: disable=pyi-error
+from grpc import _common  # pytype: disable=pyi-error
```

Comment:
I think this is related to pytype version, pytype will have this pyi-error even for files which we already added types like `_common.py` ([failed test](https://source.cloud.google.com/results/invocations/a122e7d1-2b1b-43e9-bf89-e7bc97cfa081/targets/github%2Fgrpc%2Frun_tests%2Fsanity_linux_dbg_native%2Ftools%2Fdistrib%2Fcheck_pytype.sh/tests)), I tried test it locally with 2022.6.14 version and it passed.

Since I already have an action item to update pytype version to match the version used in g3, I created an issue to track it: https://github.com/grpc/grpc/issues/31706.